plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseConnectorSmokeTest.java

```diff
@@ -53,4 +55,46 @@ public void testShowCreateTable()
                         "   engine = 'LOG'\n" +
                         ")");
     }
+
+    @Override
+    @Test
```

Comment:
nit: Replace the order for consistency with other places.

Reply:
changed